src/helpers/path.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * takes in current location and target location (eg, the location of the flag) and the map
+ * represented as a grid of 1 and 0, where 1s are traversable and 0s are not. Uses A* to calculate
+ * the best path
+ */
+export default function getTarget(myX, myY, targetX, targetY, grid) {
+  // TODO: handle edge cases regarding target and current position
+  // diagonal is true if we consider diagonal steps on the grid
+  const diagonal = false;
+  const graph = new Graph(grid, { diagonal });
+  const start = graph.grid[myX][myY];
+  const end = graph.grid[targetX][targetY];
+  // calculate shortest path list
+  const shortestPath = astar.search(graph, start, end,
+    { heuristic: diagonal ? astar.heuristics.diagonal : astar.heuristics.manhattan });
+
+  // Find the furthest cell in the direction of the next cell
+  let winner = 0;
```

Comment:
This code block is really confusing to me (i've read it over a bunch of times), however, I have a feeling it will be replaced when we integrate code for #31 , so I'm okay with it being here for now